feature_layers/service-feature-table-no-cache/README.md

```diff
@@ -1,22 +1,38 @@
-# Service Feature Table (No Cache)
+# Service feature table (no cache)
 
-Show features without caching.
+Display a feature layer from a service using the **no cache** feature request mode.
 
-On-interaction-no-cache mode will always fetch Features from the server and doesn't cache any Features on the client's side. This meaning that Features will be fetched whenever the ArcGISMap pans, zooms, selects, or queries.
+![Image of service feature table no cache](ServiceFeatureTableNoCache.png)
 
-![](ServiceFeatureTableNoCache.png)
+## Use case
 
-## How it works
+`ServiceFeatureTable` supports three request modes, which define how features are requested from the service and stored in the local table. The feature request modes have different performance characteristics. Use **no cache** in scenarios where you always want the freshest data. See [Table performance concepts](https://developers.arcgis.com/net/latest/wpf/guide/layers.htm#ESRI_SECTION1_40F10593308A4718971C9A8F5FB9EC7D) in the *ArcGIS Runtime SDK for .NET* guide to learn more.
+
+## How to use the sample
+
+Run the sample and pan and zoom around the map. With each interaction, new features will be requested from the service and displayed on the map.
 
-How to set `FeatureRequestMode.ON_INTERACTION_NO_CAHCE` mode:
+## How it works
 
-1. Create a `ServiceFeatureTable` from a URL.
-2. Set request mode of table, `ServiceFeatureTable.setFeatureRequestMode(FeatureRequestMode.ON_INTERACTION_NO_CAHCE)`.
+1. Set the `ServiceFeatureTable.FeatureRequestMode` property of the service feature table to `NO_CACHE` before the table is loaded.
+2. Add the table to the map using a `FeatureLayer`; features will be requested for the visible extent as the user pans and zooms.
 
 ## Relevant API
 
-* ArcGISMap
 * FeatureLayer
-* MapView
+* FeatureRequestMode
 * ServiceFeatureTable
-* ServiceFeatureTable.FeatureRequestMode
+
+## About the data
+
+The U.S. National Bridge Inventory describes 600,000 bridges in the United States. The sample uses [US Bridges](https://arcgisruntime.maps.arcgis.com/home/item.html?id=250b103a722c4e1ea71e562eac61be1b), a modified copy of the U.S. National Bridge Inventory hosted on ArcGIS Online. The sample opens with an initial visible extent centered over Bridgeport, CT.
```

Comment:
same as samples above, for Android we didn't change the data (open issue though) and used the following description:

> The sample opens with an initial visible extent centered over pool permit statuses in California.

Reply:
Good spot, I've updated the sample to use the data described in the readme!